scripts/Muon/FFTPresenter.py

```diff
@@ -0,0 +1,57 @@
+from __future__ import (absolute_import, division, print_function)
+from six import iteritems
+import mantid.simpleapi as mantid
+
+
+class FFTPresenter(object):
+
+    def __init__(self,view):
+        self.view=view
+        # set data
+        self.getWorkspaceNames()
+        #connect
+        self.view.tableClickSignal.connect(self.tableClicked)
+        self.view.buttonSignal.connect(self.handleButton)
+
+    # only get ws that are groups or pairs
+    # ignore raw
+    def getWorkspaceNames(self):
+        options=str(mantid.AnalysisDataService.getObjectNames()).replace(" ", "")
+        options=options[2:-2]
+        options=options.split("','")
+        final_options=[]
```

Comment:
To me it looks like `final_options` is the same as `options` here. Maybe I am missing something